@here/harp-mapview/lib/MapView.ts

```diff
@@ -1624,9 +1624,22 @@ export class MapView extends THREE.EventDispatcher {
         pitchDeg: number = 0
     ): void {
         if (this.projection.type === ProjectionType.Planar) {
+            const distanceToGround = MapViewUtils.calculateDistanceToGroundFromZoomLevel(
+                this,
+                zoomLevel
+            );
+            const pitchRad = THREE.Math.degToRad(pitchDeg);
+            const projectedDistanceOnTheGround = distanceToGround * Math.tan(pitchRad);
+
+            const yawRad = THREE.Math.degToRad(yawDeg);
+            const worldTargetCoordinates = this.projection.projectPoint(geoPos);
+
+            this.m_camera.position.set(
+                worldTargetCoordinates.x + Math.sin(yawRad) * projectedDistanceOnTheGround,
```

Comment:
Why do you move the camera position in x and y direction? This API should just set the position of the camera using the projected geoPos and adjust the height (z for planar projection) based on the provided zoomlevel.
If you want to set the target/look-at point of the camera you would not use this API.
Looking at the code below for the sphere projection, I think it would also work for the planar projection, so just remove the if part and keep the else part.

Reply:
I removed this code.
And replaced `camera.matrixWorldNeedsUpdate = true` with `camera.updateMatrixWorld()` in `zoomOnTargetPosition` method

Reply:
...and camera moves smoothly with sync rendering.